flyteidl/protos/flyteidl/core/compiler.proto

```diff
@@ -29,6 +31,12 @@ message CompiledWorkflow {
     ConnectionSet connections = 2;
 }
 
+// Output of the compilation step. This object represents one LaunchPlan. We store more metadata at this layer
+message CompiledLaunchPlan {
```

Comment:
did you mean to include CompiledLaunchPlan + LaunchPlanTemplate as part this OSS change?